tests/e2e/APNSNetworkServiceTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare( strict_types = 1 );
+
+class APNSNetworkServiceIntegrationTest extends APNSTest {
```

Comment:
Would you consider something like

```suggestion
/**
 * @group e2e
 */
class APNSNetworkServiceIntegrationTest extends APNSTest {
```

So that when developers want to run only the unit tests locally (for a faster feedback cycle with less setup involved) can run PHPUnit with the `--exclude-group e2e` option?

I'm suggesting `e2e` as the name because it's the same as the folder in which this test is.

Reply:
Sure – the test is pretty fast, but as we add more it might not be. Can't hurt!